ReleaseTooling/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -714,9 +726,17 @@ struct FrameworkBuilder {
       slicedBinariesForCarthage(fromFrameworks: slicedFrameworks,
                                 workingDir: platformFrameworksDir)
 
+    // Copy the framework in the appropriate directory structure.
```

Comment:
Refactored to build the framework in place instead of building, moving and deleting